ext_emconf.php

```diff
@@ -4,14 +4,14 @@
     'title' => 'Doctrine Migrations',
     'description' => 'Enable SQL migration powered by Doctrine',
     'category' => 'service',
-    'version' => '1.0.0',
+    'version' => '2.0.0',
```

Comment:
If it is not breaking at all, is it really a major release?

Reply:
It is more a working change ;)